fppanalysis/two_dim_velocity_estimates.py

```diff
@@ -108,8 +108,33 @@ def get_2d_velocities_from_time_delays(delta_tx, delta_ty, delta_x, delta_y):
         return delta_x / delta_tx, 0
     fx = delta_x / delta_tx
     fy = delta_y / delta_ty
+
     return fx / (1 + (fx / fy) ** 2), fy / (1 + (fy / fx) ** 2)
 
+def get_1d_velocities_from_time_delays(delta_tx, delta_ty, delta_x, delta_y):
+    """
+    Estimates radial and poloidal velocities from naive method 
+    given the input parameters:
+    Input:
+         delta_tx Estimation of the time delay between radially separated points.
+         delta_ty Estimation of the time delay between poloidally separated points.
+         delta_x Spatial separation between radially separated points.
+         delta_y Spatial separation between poloidally separated points.
+
+    These quantities should be obtained from three pixel points: a reference pixel point,
+    a pixel point separated radially, and a pixel point separated poloidally.
+    Returns:
+         vx Radial velocity
+         vy Poloidal velocity
+    """
+    if delta_tx == 0:
+        return 0, 0
+    if delta_ty == 0:
+        return 0, 0
+
+    vx = delta_x / delta_tx
+    vy = delta_y / delta_ty
+    return vx, vy 
```

Comment:
Write this method with only one return statement. 
Also, if delta_tx == 0, but delta_ty!=0 you can still return a poloidal velocity (now you return 0,0)